p2p/src/sessions/mod.rs

```diff
@@ -31,14 +31,20 @@ pub enum SessionStatus {
 
 /// Sessions struct contains:
 /// - server address used to listen to incoming connections
-/// - lists of of inbound and outbound sessions parametrized with their reference (T)
+/// - list of inbound sessions parametrized with their reference (T)
+/// - list of consolidated outbound sessions parametrized with their reference(T)
+/// - list of unconsolidated outbound sessions parametrized with their reference(T)
 pub struct Sessions<T> {
     /// Server address listening to incoming connections
     pub server_address: Option<SocketAddr>,
     /// Inbound sessions: __untrusted__ peers that connect to the server
     pub inbound_sessions: BoundedSessions<T>,
```

Comment:
the name of `xxxx_session` fields is kind of long, could we just call them `xxxx` since they are wrapped in `Session` "already"?